frontend/index.html

```diff
@@ -403,6 +404,7 @@
           // get units
           const dist_units = (data.display_config.use_imperial) ? "miles" : "kilometers";
           console.log("Units for display are", data.display_config.use_imperial, dist_units);
+          mode_studied = data.intro.mode_studied
```

Comment:
Where is this variable used? Below it looks like you just use `data.intro.mode_studied` instead of this var

Reply:
Oh wait I see it later, can we use it around line 430 too?

Reply:
Since, we are not modifying `mode_studied` in other places, we can definitely use it in line 430 too.
Also, `mode_studied` has been declared outside the `function` such that it can be utilized from other `script` block too.
Yes, updated line 430 and other places to use `mode_studied` instead of `data.intro.mode_studied`.